internal/evaluator/conftest_evaluator.go

```diff
@@ -483,6 +483,37 @@ func createConfigJSON(ctx context.Context, dataDir string, p policy.Policy) erro
 	return nil
 }
 
+// addRuleData creates the $dataDir/rule_data__configuration__/ruledata.json
+// files with the provided custom rule data for each source in the data
+// directory
+func addRuleData(ctx context.Context, dataDir string, p policy.Policy) error {
+	if p == nil {
```

Comment:
What would call this function with a nil policy?

Reply:
No idea, copied from `createConfigJSON`